<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="puja" failures="2" tests="8" name="Test" time="0.159" errors="0" timestamp="2021-10-30T09:24:14 IST">
  <testcase classname="com.testng.DemoTest" name="Demo" time="0.017"/>
  <testcase classname="com.testng.AssertionInTestNg" name="Test1" time="0.009"/>
  <testcase classname="com.testng.AssertionInTestNg" name="Test2" time="0.002"/>
  <testcase classname="com.testng.FirstTestNGScript" name="checkOut" time="0.002"/>
  <testcase classname="com.testng.FirstTestNGScript" name="loginApplication" time="0.002"/>
  <testcase classname="com.testng.FirstTestNGScript" name="selectItems" time="0.005"/>
  <testcase classname="com.testng.AssertionTestNG" name="Test1" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [13] but found [12]">
      <![CDATA[java.lang.AssertionError: expected [13] but found [12]
at com.testng.AssertionTestNG.Test1(AssertionTestNG.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase classname="com.testng.AssertionTestNG" name="Test2" time="0.006">
    <failure type="java.lang.AssertionError" message="Dropdown count does no match please check once expected [13] but found [12]">
      <![CDATA[java.lang.AssertionError: Dropdown count does no match please check once expected [13] but found [12]
at com.testng.AssertionTestNG.Test2(AssertionTestNG.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test2 -->
</testsuite> <!-- Test -->
